activerecord/test/cases/adapters/postgresql/network_test.rb

```diff
@@ -93,4 +93,10 @@ def test_schema_dump_with_shorthand
     assert_match %r{t\.cidr\s+"cidr_address",\s+default: "192\.168\.1\.0/24"}, output
     assert_match %r{t\.macaddr\s+"mac_address",\s+default: "ff:ff:ff:ff:ff:ff"}, output
   end
+
+  def test_mac_address_change_case_does_not_mark_dirty
+    model = PostgresqlNetworkAddress.create(mac_address: "Ab:Cd:Ef:01:02:03")
+    model.mac_address = model.mac_address.swapcase
+    assert_not model.changed?
```

Comment:
```suggestion
    assert_not_predicate model, :changed?
```

This produces better output when the test fails.

Reply:
Updated